### ISSUE TITLE
Validating with custom javascript rule

### ISSUE THREAD

**Issue by clarkewing** [proengsoft/laravel-jsvalidation#241]
I use [Zxcvbn](https://github.com/bjeavons/zxcvbn-php) to give users an idea of how strong their password is, and I'd also like to enforce a minimum password strength using the score provided by Zxcvbn.

So I built a custom method for jQuery validation:
```
$.validator.addMethod('passwordMinScore', function (value, element, params) {
    return this.optional(element) || $(element).attr("data-score") >= params[0]; // data-score is set by function which is triggered on key-up event
}, "The password you entered is not strong enough.");
```
Up to there everything works fine.
But then I tried adding the rule to the password field:
```
$('#password').rules('add', {
    passwordMinScore: 2
});
```
When doing so, I get the following error: `Uncaught TypeError: Cannot read property 'settings' of undefined`, which ties into line 133 of jquery.validate.js: `settings = $.data( element.form, "validator" ).settings;`.

Am I doing this wrong? How can I add a custom javascript rule to a field? It's not validated on the server.

**Comment by waterloomatt** [comment]
Looks like a timing issue. Wrapping this in document.ready works:

    ```
$(document).ready(function () {
            $.validator.addMethod('passwordMinScore', function (value, element, params) {
                return false;
            }, "The password you entered is not strong enough.");

            $('#title').rules('add', {
                passwordMinScore: 2
            });
        });
```

**Comment by bytestream** [comment]
Due to the volume of issues I'm afraid I'm closing this issue due to lack of activity. This avoids fellow collaborators wasting time investigating this issue, and also prevents giving a false impression that it will eventually be addressed.

To the reporter, please reopen if you still encounter this on the latest version providing a minimal test case. You can also contribute directly by submitting a PR. 

Thanks.
